bitnami/rabbitmq/README.md

```diff
@@ -263,35 +263,31 @@ extraConfiguration: |
   log.console.formatter = json
 ```
 
-## How to Avoid Deadlocked Deployments After a Cluster-Wide Restart
+### How to Avoid Deadlocked Deployments After a Cluster-Wide Restart
```

Comment:
This deserves a level 2 header because it is a crucially important problem and not just "a yet another configuration topic", as regularly appearing discussions and issues suggest.